### ISSUE TITLE
Connecting to Gitlab via SSH through VPN

### ISSUE THREAD

**Issue by jbgruenwald** [cpoetter/Synology-GitLab-Setup#2]
Hi,
thanks for this great repo. It helped a lot in configuring my Gitlab. Currently I have a problem with connecting to Gitlab when I use the L2TP VPN server from Synology. Does anybody know if there is a firewall that blocks SSH from the VPN? If yes, where can I configure an exception rule?

**Comment by cpoetter** [comment]
Hi @jbgruenwald!

I use GitLab through the L2TP VPN from my Mac as well. Works without a problem. I do not remember that I had to configure anything to make this possible. Did you check the `Send all traffic over VPN connection` checkbox?

**Comment by jbgruenwald** [comment]
I'm not sure, where I can find this checkbox. In the VPN Server config I only have the following options:
![L2TP-config](https://user-images.githubusercontent.com/23033344/56654501-85f10600-6690-11e9-83a5-7f1b5774f416.png)

Or is it in the VPN client? I didn't find it there too. It looks like only ssh doesn't work, because I get a connection via http in my browser. But git always says, it can't find the host, when I try to push my code. Telnet also doesn't find an open socket via port 30001. But with ping I get a reply. That's very confusing. Do you have an idea, what could be the problem?

**Comment by cpoetter** [comment]
I meant on the client. On the Mac it is in the VPN settings under `Advanced...` -> `Options`.

**Comment by jbgruenwald** [comment]
Ok, thanks for the idea. For the VPN connection I use network manager in ubuntu and I can't find a checkbox like this. But I found something else: The problem has to do something with DNS. I tried telnet using the IP-Adress and it worked. Even git works now, but I can't use hostnames. For now that's ok, but I'll have a look into that later. Thanks anyway for the help and I think, the issue can be closed.
